librz/include/rz_il/definitions/sort.h

```diff
@@ -31,6 +32,9 @@ typedef struct rz_il_sort_pure_t {
 		struct {
 			ut32 length;
 		} bv;
+		struct {
+			RzFloatFormat format;
```

Comment:
There must also be a length like in bitvector in here, right?

Reply:
Not sure. For me `format` indicates the bitvector length implicitly, should we set an length here ?

Reply:
I see, if the format defines the length unambiguously then we should not have length it here.